spec/controllers/bookings_spec.rb

```diff
@@ -0,0 +1,113 @@
+require 'rails_helper'
+require 'auth_helper'
+
+RSpec.describe BookingsController, type: :controller do
+  let(:role) { create(:custom_role) }
+  let!(:user) { FactoryBot.create(:user, role_id: role.id, name: "abc") }
+  let(:token_new) { encode_token(id: user.id) }
+  let(:token) { "Bearer #{token_new}" }
+  let!(:venues) { FactoryBot.create_list(:venue, 3) }
+  let(:venue) { venues.first }
+  let!(:booking) { FactoryBot.create(:booking, user: user, venue: venue) }
+  let(:valid_params) {
+    {
+      user_id: user.id,
+      venue_id: venue.id,
+      booking_date: Faker::Date.forward(days: 30),
+      start_time: Faker::Time.between_dates(from: Date.today, to: Date.today + 30.days, period: :day),
+      end_time: Faker::Time.between_dates(from: Date.today, to: Date.today + 30.days, period: :day),
+      status: STATUSES.sample
+    }
+  }
+
+  describe 'GET #index' do
+    context 'booking of a particular user' do
+      it 'renders JSON with email' do
+        request.headers["Authorization"] = token
+        get :index, params: { user_id: user.id }
+        expect(response).to have_http_status(:ok)
```

Comment:
where is json comparsion?
how renders json with email?

excpect response should have email?